exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -473,7 +473,14 @@ func (ku *Kucoin) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (
 			Currencies: balances,
 		})
 	case asset.Spot, asset.Margin:
-		accountH, err := ku.GetAllAccounts(ctx, "", ku.accountTypeToString(assetType))
+		assetTypeString := "main"
+		switch assetType {
+		case asset.Spot:
+			assetTypeString = SpotTradeType
+		case asset.Margin:
+			assetTypeString = "margin"
+		}
+		accountH, err := ku.GetAllAccounts(ctx, currency.EMPTYCODE, assetTypeString)
```

Comment:
<img width="1334" alt="image" src="https://github.com/user-attachments/assets/c678115d-0e41-4cf1-be79-1baa03d1485a">
This is incorrect and blocks auth usage